src/Accounts/Accounts/Utilities/InteractiveSubscriptionSelectionHelper.cs

```diff
@@ -0,0 +1,176 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Common.Exceptions;
+using Microsoft.Azure.Commands.Profile.Properties;
+using Microsoft.WindowsAzure.Commands.Common;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Profile.Utilities
+{
+    internal static class InteractiveSubscriptionSelectionHelper
+    {
+        private static string DefaultSubscriptionMark = $"{PSStyle.ForegroundColor.White}*{PSStyle.Reset}";
```

Comment:
We should define constants as const if possible. This comment also applies to the magic numbers in this type

Reply:
not possible for an interpolate string. Marked magic number as const.